### DISCUSSION TITLE
go-getter v1.7.0 presents as a virus

### DISCUSSION THREAD

**Discussion by commodore-84** [aquasecurity/trivy#5358] [Bugs]
### Description

when running
go install github.com/aquasecurity/tfsec/cmd/tfsec@latest
this attempts to download go-getter v1.7.0 from https://proxy.golang.org/github.com/hashicorp/go-getter/@v/v1.7.0.zip and this is presenting as a virus.

Has anyone else experienced this?

Could go-getter be upgraded to 1.7.1 or 1.7.2 in the tfsec go.mod file?

### Desired Behavior

no errors seen when performing a go install

### Actual Behavior

go: downloading github.com/hashicorp/go-getter v1.7.0
../../../go/pkg/mod/github.com/aquasecurity/defsec@v0.84.1/pkg/scanners/terraform/parser/resolvers/remote.go:11:2: github.com/hashicorp/go-getter@v1.7.0: reason=Trojan
reasoncode=AV_AUTHENTIUM_VIRUS_SW_MW_BLOCK

### Reproduction Steps

```bash
1.go install github.com/aquasecurity/tfsec/cmd/tfsec@latest
2.
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A
```

### Operating System

WSL Ubuntu 22.04.3 LTS

### Version

```bash
N/A
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by nikpivkin** [comment]
Hi @el-chazmo ! 

tfsec is now part of Trivy: https://github.com/aquasecurity/tfsec/discussions/1994.
You can read the [migration guide](https://github.com/aquasecurity/tfsec/blob/master/tfsec-to-trivy-migration-guide.md).
